src/pages/index.js

```diff
@@ -94,8 +101,35 @@ console.log(currentUserInfo.getUserInfo());
 const profileEditModalPopup = new PopupWithForm(
   "#modal_profile-edit",
   (data) => {
-    currentUserInfo.setUserInfo(data.title, data.description);
-    profileEditModalPopup.close();
+    //currentUserInfo.setUserInfo(data.title, data.description);
+    //profileEditModalPopup.close();
+    //profileEditModalPopup.updating();
+    profileEditModalPopup.setSaveButtonText("Updating");
+
+    api
+      .editCurrentUser("/users/me", data)
+      .then(profileEditModalPopup.close())
```

Comment:
Needs correction

1) `then` method accepts a function
2) you should update the info and close the popup using one `then` block

```js
.then(() => {
 // update the info
 // close the popup
})
```

(fix that everywhere)

Reply:
Done